### PR TITLE
Add r-latex2exp

### PR DESCRIPTION
**Pull request by apcamargo**
The recipe was created using `conda_r_skeleton_helper`.

### CODE REVIEW

File: recipes/r-latex2exp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - r-stringr
+
+test:
+  commands:
+    - $R -e "library('latex2exp')"           # [not win]
+    - "\"%R%\" -e \"library('latex2exp')\""  # [win]
+
+about:
+  home: http://github.com/stefano-meschiari/latex2exp
+  license: MIT
```
Comment (grst) [inline recipes/r-latex2exp/meta.yaml]: please add a `license_file`

Comment (apcamargo) [reply recipes/r-latex2exp/meta.yaml]: Done!

**Comment by grst** [comment]
@conda-forge/staged-recipes, LGTM now

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**